common/lib/xmodule/xmodule/tests/test_xblock_wrappers.py

```diff
@@ -290,6 +290,7 @@ def test_leaf_node(self, cls_and_fields):
         # pylint: disable=no-member
         descriptor.runtime.id_reader.get_definition_id = Mock(return_value='a')
         descriptor.runtime.modulestore = modulestore
+        descriptor._xmodule.graded = 'False'
```

Comment:
Update: I talked to @nasthagiri briefly about this. She assured me that "graded" is widely assumed to exist on all capa problems, and her preference is that we do not litter the code with conditionals (that could hide actual production issues). I am convinced. :)